odl/tomo/backends/astra_cpu.py

```diff
@@ -191,6 +197,18 @@ def astra_cpu_backward_projector_call(proj_data, geometry, reco_space,
     # Run algorithm and delete it
     astra.algorithm.run(algo_id)
 
+    # Fix scaling issue
+    # parallel2d and fanflat do scale quadratically with linear voxel size
+    # and linearly with inverse pixel size
```

Comment:
Should be clearer. That sentence changed a few times until I figured out how the rescaling should be done.